### DISCUSSION TITLE
Desktop shortcut does not start if automatic cache cleaning was installed

### DISCUSSION THREAD

**Discussion by Bad-Skiddy** [SpotX-Official/SpotX#203] [Question]
So I'm wondering why this has happened?

![image](https://user-images.githubusercontent.com/108899110/200597634-5c61063d-9ceb-4dd4-afa5-440178d0e9c0.png)

Can anybody tell me the correct path/program?

**Comment by amd64fox** [comment]
in case you want to examine the code inside `hide_window.vbs`
using notepad or any other text editor

or does it happen after you try to launch Spotify ?

**Reply by NeuroCPP** [reply]
had happened to me just restarted and fixed the issue first time
but next time i just removed shortcut

**Reply by Bad-Skiddy** [reply]
> in case you want to examine the code inside `hide_window.vbs` using notepad or any other text editor
> 
> or does it happen after you try to launch Spotify ?

yeah, it happens when I try to launch from the shortcut.

**Reply by amd64fox** [reply]
you just had to find the `microsoft windows based script host` application in the list and check the box to always open it with it
![Снимок экрана 2022-11-08 205415](https://user-images.githubusercontent.com/62529699/200639446-1f72918d-c2fe-446e-97f1-a5664db8b95f.jpg)

**Reply by amd64fox** [reply]
Usually the OS does this automatically, but apparently not always.

**Reply by Bad-Skiddy** [reply]
Yeah, I still can't find this. do you the path? or another way to run a vbs file?

**Reply by Bad-Skiddy** [reply]
![image](https://user-images.githubusercontent.com/108899110/200650724-99169164-e335-48a5-ba75-8fee4e8a5cc9.png)

I managed to get it to work just by manually selecting the .bat file.

**Reply by amd64fox** [reply]
this is not quite the right launch logic, vbs is used here to hide the cmd window.

Try manually specifying the vbs address `%WinDir%\System32\wscript.exe`

![Анимация](https://user-images.githubusercontent.com/62529699/200660920-478b7160-e0eb-4f2b-9d83-dfc42353852e.gif)
